### ISSUE TITLE
Various bugs in 12/05 session

### ISSUE THREAD

**Issue by moufmouf** [thecodingmachine/workadventure#105]
Seen in today's session.

I don't know if this is related (probably not)

Logs on the back:
```
client doesn't exist for  778efc7d-e79b-43b7-8190-d4f887d00f5a
client doesn't exist for  778efc7d-e79b-43b7-8190-d4f887d00f5a
client doesn't exist for  778efc7d-e79b-43b7-8190-d4f887d00f5a
client doesn't exist for  778efc7d-e79b-43b7-8190-d4f887d00f5a
client doesn't exist for  778efc7d-e79b-43b7-8190-d4f887d00f5a
```

Empty circles:

![image](https://user-images.githubusercontent.com/1290952/81711960-6c298900-9474-11ea-8340-d5e050259ea3.png)

Note that I don't know if the circles are still existing on the server side.

**Comment by gulien** [comment]
Related #104

**Comment by moufmouf** [comment]
Closed by #120 (hopefully!)
